JustBurp/myapp/views/partials/chefDailyTimeSetupCard.hbs

```diff
@@ -7,58 +7,99 @@
+            $(freeItemInputNode).prop("disabled", isDisable);
         });
     });
 </script>
 
 <div class="timeSetupCard">
-    <div>
-        <div class="date"><b>{{dateString}}</b></div>
-        <div class="noCookingCheckbox">
-            <input type="checkbox" {{#if isNoCooking}}checked{{/if}} /> No cooking
-        </div>
-    </div>
-    <div>
-        Prepare:
+    <div class="noCookingCheckbox">
+        <input type="checkbox" {{#if isNoCooking}}checked{{/if}} /> No cooking
     </div>
+    <div class="subHeader">Taking Order Deadline:</div>
     <div class="prepareTimeBox">
-        From
-        <input id="prepareStartTime" type="text" class="time ui-timepicker-input" name="prepareStartTime" value={{prepareStartTime}}
-            {{#if isNoCooking}}disabled{{/if}} />
-    </div>
-    <div>
-        Deliver:
+        <div>From Now to</div>
+        <div>
+            <input type="text" class="time ui-timepicker-input prepareStartTime"
+                   name="prepareStartTime" value={{prepareStartTime}}
+                {{#if isNoCooking}}disabled{{/if}} />
+        </div>
     </div>
+    <div class="subHeader">Deliver:</div>
     <div>
         <div class="col-sm-6">
-            From
-            <input id="startTime" type="text" class="time ui-timepicker-input" name="startTime" value={{startTime}}
-                {{#if isNoCooking}}disabled{{/if}} />
+            <div>From</div>
+            <div>
+                <input type="text" class="time ui-timepicker-input startTime" name="startTime" value={{startTime}}
+                    {{#if isNoCooking}}disabled{{/if}} />
+            </div>
         </div>
-        <div class="col-sm-6">
-            To
-            <input id="endTime" type="text" class="time ui-timepicker-input" name="endTime" value={{endTime}}
-                {{#if isNoCooking}}disabled{{/if}} />
+        <div class="col-sm-6 endTimeContainer">
+            <div>To</div>
+            <div>
+                <input type="text" class="time ui-timepicker-input endTime" name="endTime" value={{endTime}}
+                    {{#if isNoCooking}}disabled{{/if}} />
+            </div>
         </div>
     </div>
+
+    <div class="noFreeItemCheckbox">
+        <input type="checkbox" {{#if isNoFreeItem}}checked{{/if}} {{#if isNoCooking}}disabled{{/if}} /> No free items
+    </div>
+    <div class="subHeader freeItemSubHeader">Free Item Per Order:</div>
+    <div class="freeItemContainer">
+        <div class="col-sm-7">{{> qtyChange applyStyling=true isDisabled=isDisabled}}</div>
+        <div class="col-sm-5"><input type="text" class="freeItemTextInput" name="freeItemTitle" placeholder="Free item title"
+            {{#if isDisabled}}disabled{{/if}} value={{freeItemTitle}} ></div>
+    </div>
+
     <div>
         <input type="hidden" name="isNoCookingChecked" value={{isNoCooking}} />
     </div>
+    <div>
+        <input type="hidden" name="isNoFreeItemChecked" value={{isNoFreeItem}} />
+    </div>
 </div>
```

Comment:
Should we have another function here to check whether or not endTime is at least one hour later than the startTime if we want to check this logic in the font-end?